document/src/vespa/document/base/fieldpath.cpp

```diff
@@ -141,9 +142,9 @@ FieldPathEntry::parseKey(std::string_view & key)
     vespalib::string v;
     const char *c = key.data();
     const char *e = c + key.size();
-    for(;(c < e) && isspace(c[0]); c++);
+    for(;(c < e) && std::isspace(static_cast<unsigned char>(c[0])); c++);
     if ((c < e) && (c[0] == '{')) {
-        for(c++;(c < e) && isspace(c[0]); c++);
+        for(c++;(c < e) && std::isspace(static_cast<unsigned char>(c[0])); c++);
```

Comment:
perhaps an utility method similar to

const char * eatWhite(const char * s, const char * e) {
    for(;(c < e) && std::isspace(static_cast<unsigned char>(c[0])); c++);
    return c;
}